packages/rspack/tests/__snapshots__/StatsTestCases.test.ts.snap

```diff
@@ -1072,28 +1072,26 @@ exports[`StatsTestCases should print correct stats for identifier-let-strict-mod
 {
   "errors": [
     {
-      "formatted": "error[javascript]: JavaScript parsing error
-  ┌─ tests/statsCases/identifier-let-strict-mode/index.js:1:5
-  │
-1 │ let let = let;
-  │     ^^^ \`let\` cannot be used as an identifier in strict mode
-2 │ 
-
+      "formatted": "  × Error[javascript]: JavaScript parsing error
+   ╭─[tests/statsCases/identifier-let-strict-mode/index.js:1:1]
```

Comment:
position looks wrong.

Reply:
This seems another limitation of miette. Check out oxc's snaps, for example, this one: https://github.com/oxc-project/oxc/blob/6e5aad26393c363e64cd3040547f3184538fa9b7/crates/oxc_linter/src/snapshots/adjacent_overload_signatures.snap. It only resolves to line instead of column.

Reply:
ok